### ISSUE TITLE
Broken Link for  "Getting Started Guide" 

### ISSUE THREAD

**Issue by markgraf** [debops/debops-tools#195]
The link to the "Getting Started Guide" (https://docs.debops.org/en/latest/debops/docs/getting-started.html) is broken: "Sorry, this page does not exist yet" 

I guess the link should point to...

https://docs.debops.org/en/latest/debops-playbooks/docs/guides/getting-started.html

instead?

**Comment by drybjed** [comment]
Yes, you're correct. Do you want to create a PR that fixes this?

**Comment by markgraf** [comment]
Done :-)

**Comment by ypid** [comment]
Thanks for fixing this in #196. Unfortunately, the URL changed again. No need to fix this as this repo is only kept for the issues. Closing so that this repo can be archived eventually. @drybjed Please close and potentially give me permission on the this repo.
